tests/backup/backup_helper.go

```diff
@@ -179,6 +179,63 @@ const (
 	Parallel
 )
 
+var (
+	// AppRuleMaster is a map of struct for all the value for rules
+	// This map needs to be updated for new applications as and whe required
+	AppRuleMaster = map[string]backup.AppRule{
+		"cassandra": {
+			PreRule: backup.PreRule{
+				Rule: backup.RuleSpec{
+					ActionList: []string{"nodetool flush -- keyspace1;", "echo 'test"}, PodSelectorList: []string{"app=cassandra", "app=cassandra1"}, Background: []string{"false", "false"}, RunInSinglePod: []string{"false", "false"}, Container: []string{},
```

Comment:
Isn't this PodSelectorList coming from "AppParameters" defined portworx.go ?

Reply:
It is not coming from "AppParameters" defined portworx.go

As mentioned in description, once https://portworx.atlassian.net/browse/PA-1427 is merged, there will be only AppRuleMaster and AppParameter will be discarded.